src/App.jsx

```diff
@@ -14,45 +16,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_ALPHABETICALLY = 'Alphabet';
+const SORT_BY_LENGTH = 'Length';
+const SORT_REVERSE = false;
+
+function getPreparedGoods(goods, { sortType, reverse }) {
+  const preparedGoods = [...goodsFromServer];
```

Comment:
You are using the 'goodsFromServer' array directly. Instead, use the 'goods' parameter passed to the 'getPreparedGoods' function to ensure the function is reusable and doesn't depend on outer scope variables.